Documentation/blame-options.adoc

```diff
@@ -83,6 +83,7 @@ include::line-range-format.adoc[]
 	or `--incremental`.
 
 -M[<num>]::
+--find-renames[=<num>]::
 	Detect moved or copied lines within a file. When a commit
```

Comment:
No, you cannot do that. The existing `-M` option is not about whole-file rename detection, therefore we cannot reuse that, and my instructions regarding `-M` are moot.

You have to _introduce_ a new `--find-renames` option instead.